warp10/src/main/java/io/warp10/continuum/MetadataUtils.java

```diff
@@ -67,10 +72,23 @@ public class MetadataUtils {
 
     //
     // Ensure that the limits will allow for serialized metadata to fit in FoundationDB
+    // This is checked for all versions of Warp 10 since they could either store directly
+    // in FoundationDB (standalone+ / distributed) or have a datalog replication in place
+    // to another instance which could be using FoundationDB.
+    // The check can be relaxed in the case of a standalone version by setting the config
+    // key 'warp.relax.metadata.limit' to 'true'
     //
 
     if (MAX_CLASS_SIZE + MAX_LABELS_SIZE + MAX_ATTRIBUTES_SIZE > FDBUtils.MAX_VALUE_SIZE - FDB_SIZE_RESERVED) {
-      throw new RuntimeException("Invalid metadata limits, '" + Configuration.WARP_CLASS_MAXSIZE + "' + '" + Configuration.WARP_LABELS_MAXSIZE + "' + '" + Configuration.WARP_ATTRIBUTES_MAXSIZE + "' should be less than " + (FDBUtils.MAX_VALUE_SIZE - FDB_SIZE_RESERVED));
+      if (!WarpConfig.isStandaloneMode() || Constants.BACKEND_FDB.equals(WarpConfig.getProperty(Configuration.BACKEND)) || !"true".equals(WarpConfig.getProperty(Configuration.WARP_RELAX_METADATA_MAXSIZE, "false"))) {
```

Comment:
condition problem, when limits are higher, it fails with `Invalid metadata limits, 'warp.class.maxsize' + 'warp.labels.maxsize' + 'warp.attributes.maxsize' should be less than 98976 to ensure compatibility with FoundationDB based instances` message whatever relax configuration value is.